cosmo_tester/framework/test_hosts.py

```diff
@@ -697,12 +701,18 @@ def create(self):
                 self._logger.error('Error on terraform destroy: %s', ex)
             raise
 
-    def destroy(self):
-        """Destroys the OpenStack infrastructure."""
-        self._logger.info('Destroying test hosts..')
-        with self._tmpdir:
-            self._terraform.destroy(
-                    ['-var-file', self._terraform_inputs_file, '-force'])
+    def destroy(self, err_raised=False):
+        """Destroys the OpenStack infrastructure.
+
+        :param err_raised: whether an error was raised.
```

Comment:
Add a little more info for the purpose of this flag, and lose the default because you will always have a value there

Reply:
Added.